app/Http/Controllers/PrioController.php

```diff
@@ -0,0 +1,473 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\{AuditLog, Character, Guild, Instance, Item, Raid};
+use Auth;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+
+class PrioController extends Controller
+{
+    const MAX_PRIOS = 10;
+
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->middleware(['auth', 'seeUser']);
+    }
+
+        /**
+     * Show the mass input page
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function chooseRaid($guildSlug, $instanceSlug)

```

Comment:
We should align on a naming pattern for these function names. Its hard to understand what they are doing at first without reading what its returning. 

Example: showChooseRaidView($guildSlug, $instanceSlug) or chooseRaidView($guildSlug, $instanceSlug)

Reply:
100% agree. You can put forth a spec, or I'll write something up before I write my next function (hopefully). Ideally, this spec will be combined with a PR to refactor existing functions.